Jenkinsfile

```diff
@@ -25,7 +25,7 @@ pipeline {
         maven('maven')
         git('git')
         ant('Ant')
-        jdk('JDK11')
+        jdk('OpenJDK11')

```

Comment:
I can't add the comment on line 97 where it belongs, but you forgot two spots (please re-test with another build once changed)